src/api/physicalPersonService.ts

```diff
@@ -0,0 +1,47 @@
+import { PHYSICAL_PERSON_NEW_FAIL } from '../components/shared/messages/physical-person'
```

Comment:
Mesma coisa aqui, checar https://docs.nestjs.com/exception-filters#built-in-http-exceptions e eusar as exceçoes nativas